nominees/digital-health-platform.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "Digital Health Platform",
+  "aliases": [
+    "DHP"
+  ],
+  "description": "The Digital Health Platform is an end-to-end electronic medical records platform designed to support patient data collection, health records management, storage and reporting.",
+  "website": "http://kenyadhp.org",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://www.apache.org/licenses/LICENSE-2.0"
+    },
+    {
+      "spdx": "MS-PL",
+      "licenseURL": "https://opensource.org/licenses/MS-PL"
+    },
+    {
+      "spdx": "PostgreSQL",
+      "licenseURL": "https://www.postgresql.org/about/licence/"
+    },
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://www.postgresql.org/about/licence/"
+    },
+    {
+      "spdx": "AGPL-3.0",
+      "licenseURL": "https://www.postgresql.org/about/licence/"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "This platform promotes efficient and optimized management of patients clinical data within a public health facility"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceText": "The DHP promotes the use of innovative technologies towards establishing an efficient platform for the management of patient data which includes access to health information"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositories": [],
```

Comment:
This submission is missing links to source code which anyone can view, copy, modify and share, and distribute the code in public repositories. Please contact them to provide the relevant repos. Thanks